crates/sui-graphql-rpc/src/context_data/sui_sdk_data_provider.rs

```diff
@@ -314,18 +311,6 @@ impl DataProvider for SuiClient {
         Ok(connection)
     }
 
-    async fn fetch_tx(&self, digest: &str) -> Result<Option<TransactionBlock>> {
```

Comment:
Out of interest, how come this PR removes this function?

Reply:
@oxade implemented a DataLoader for transactions and replaced fetch_tx usage, but I suppose that doesn't mean we need to remove this fetch_tx fn